templates/footer.php

```diff
@@ -1,5 +1,6 @@
 <script src="/cafeteria/js/jquery.js"></script>
 <script src="/cafeteria/js/bootstrap.bundle.js"></script>
+<script src="/cafeteria/js/admin/main.js"></script>
```

Comment:
You should not add the script file here add in your page using the `$PAGE_SCRIPTS` variable.